test/utils/xcm.ts

```diff
@@ -510,6 +510,60 @@ export class XcmFragment {
         return this;
     }
 
+    deposit_asset_definite(
+        location: any,
+        amount: bigint,
+        beneficiary: `0x${string}`,
+        network: XcmV3JunctionNetworkId["type"] | null = null
+    ): this {
+        if (hexToU8a(this.config.beneficiary).length === 20) {
```

Comment:
What's the difference between `this.config.beneficiary` and `beneficiary` arg? Inside the if only `beneficiary` is used, `this.config.beneficiary` is unused.